modules/devguide/examples/go/slow-operations.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"github.com/couchbase/gocb/v2"
+	"time"
+)
+
+func main() {
+	// #tag::config[]
+	tracerOpts := &gocb.ThresholdLoggingOptions{
+		Interval:   10 * time.Second,
+		SampleSize: 10,
+	}
+	tracer := gocb.NewThresholdLoggingTracer(tracerOpts)
+
+	opts := gocb.ClusterOptions{
+		Authenticator: gocb.PasswordAuthenticator{
+			Username: "Administrator",
+			Password: "password",
+		},
+		Tracer: tracer,
+	}
+	// #end::config[]
+	throwaway(opts)
```

Comment:
Looks like the example is failing to build because of this line?
https://github.com/couchbase/docs-sdk-go/runs/5405385655?check_suite_focus=true#step:4:99

Doesn't seem like the function exists? Or is it some new keyword maybe?

Reply:
It's a combination of how my IDE is setup and how the examples are run. My IDE really doesn't like having throwaway declared twice within the module.